src/librustdoc/html/render.rs

```diff
@@ -1020,6 +1028,18 @@ fn write(dst: PathBuf, contents: &[u8]) -> Result<(), Error> {
     Ok(try_err!(fs::write(&dst, contents), &dst))
 }
 
+fn write_minify(dst: PathBuf, contents: &[u8], enable_minification: bool) -> Result<(), Error> {
+    if enable_minification {
+        if let Ok(s) = String::from_utf8(contents.to_vec()) {
```

Comment:
Use `include_str!` instead of `include_bytes!` for the static files to avoid this UTF-8 check.

Reply:
Thanks!